src/react-app-env.d.ts

```diff
@@ -1 +1,7 @@
 /// <reference types="react-scripts" />
+
+export interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
```

Comment:
```suggestion
```
Move your custom type declaration to separate file in `types` folder, for example.
This file is react-special and it's unsafe to modify it.